### PR TITLE
Fix various search issues

### PR DESCRIPTION
**Pull request by piazzatron**
### Description
Fixes a bunch of things:
- Adds badges in autocomplete search
- Removes %20 in search bar if you search for multiple words (pressing enter, full search)
- Stops sorting exact handle matches to top on client, since DP is smart now
- Fix bug where if you hover over an autocomplete option and then press arrow key down, two opts show selected (go see this one on prod)
- Fix bug where a 5px dropshadow from the header would steal focus as you scrolled through autocomplete results (a @forrestb find)

Had to break up the selectors for badge stuff a bit, but no logic changes.

### Dragons

Nah

### How Has This Been Tested?

Locally against DP1

### CODE REVIEW

File: src/components/general/Page.module.css [hunk 1]
```diff
@@ -39,6 +39,8 @@
 
 .frosted {
   backdrop-filter: blur(10px);
+  position: relative;
+  z-index: 10;
```
Comment (raymondjacobson) [inline src/components/general/Page.module.css:43]: why did we need this?

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/piazz-client-search-improvements

File: src/containers/search-bar/SearchBar.js [hunk 2]
```diff
@@ -60,11 +61,12 @@ class SearchBar extends Component {
       this.setState({ value: '' })
       return
     }
+    const decodedValue = decodeURIComponent(value)
```
Comment (raymondjacobson) [inline src/containers/search-bar/SearchBar.js:64]: we should make sure to test searches that intentionally include a % sign, etc. as well as tag search

**Review by raymondjacobson — APPROVED** [review]
let's merge tmw after release, but generally looks great!! would love to play around on staging w/ it

nice putting up with antd

File: src/components/general/Page.module.css (hunk 1 above)
Comment (piazzatron) [reply src/components/general/Page.module.css:43]: should have been called out in the commit message for this one. It was this crazy thing where we had a 5 pixel bar that stole focus as you hovered down autocomplete results

**Pull request merged**